content/applications/services/field_service/worksheets.rst

```diff
@@ -0,0 +1,68 @@
+==========
+Worksheets
+==========
+
+**Worksheets** help your field service workers perform and report their on-site tasks. They can
+feature various information, such as instructions, to-do lists, etc. You can also format your
+worksheet using checkboxes, bullet points, blank fields to fill in, HTML, and add files, images,
+links, and more.
+
+It is common for businesses to have their workers perform the same type of field service
+repeatedly. Making custom **worksheet templates** eliminates the need to recreate the same
+worksheet each time you plan a similar field service task.
+
+Configuration
+=============
+
+To use worksheets in Field Service, go to :menuselection:`Field Service --> Configuration -->
+Settings`, enable the :guilabel:` Worksheets` feature, and click :guilabel:`Save`.
+
+.. warning::
+   Worksheet templates are designed using **Studio** and clicking :guilabel:`Design Template`
+   automatically installs the **Studio** app, which impacts your price plan.
+
+Create a worksheet template
+---------------------------
+
+To create your **worksheet templates**, go to :menuselection:`Field Service --> Configuration -->
+Worksheet Templates`. Click :guilabel:`New` and give your worksheet template a name. You can also
+add a company and a color to add clarity if needed and manually :guilabel:`Save`. Then click
+:guilabel:`Design Template` to open **Studio** and customize your worksheet template.
+
+In Studio click :guilabel:`Add`. Drag and drop the items that you want from the left column
+into your worksheet. To rearrange the items, just drag and drop them in the order that you want.
+Click on the items you added to your worksheet to customize their properties.
+
+When your Worksheet Template is complete, click :guilabel:`Close` in the top right corner of the
+page to leave **Studio**.
+
+.. seealso::
+   - :doc:`Fields and widgets in Studio <../../studio/fields>`
+
+Add a worksheet template to a Field Service task
+================================================
+
+Go to your field service task and select a :guilabel:`Worksheet Template` and click
+:guilabel:`Save`.
+
+By default, the :guilabel:`Default worksheet` template is selected. To define another default
+worksheet template, click the :guilabel:`➔` (:guilabel:`Internal link`) icon that appears when you
```

Comment:
I am pretty sure it was an arrow before but now I see this instead (this means we would need to change the screenshot as well ...)
![image](https://github.com/odoo/documentation/assets/128469033/10373c4a-b7e7-4c50-a8fb-3ca69172c692)